external/source/shellcode/osx/stager/main.c

```diff
@@ -341,152 +371,136 @@ int main(int argc, char** argv)
     }
     NSCreateObjectFileImageFromMemory_func = find_symbol(dyld, "_NSCreateObjectFileImageFromMemory", offset);
   }
-#ifdef DEBUG
   print("good symbol!\n");
-#endif
 
   // gDyld is a special struct that libdyld.dylib uses to interface with dyld4.
   // gDyld is not present in dyld3 and back.
   void* gDyld = find_symbol(dyld, "__ZN5dyld45gDyldE", offset);
   //printf("gDyld: %lld\n", gDyld);
   void * addr_main = 0;
   if (gDyld != 0) {
-#ifdef DEBUG
     print("gDyld found, using dual hijack technique.\n");
```

Comment:
It looks like I'm getting a bunch of extra junk characters printed out here when running the compiled version under Rosetta with `osx/x64/meterpreter/reverse_tcp`:

```
% ./x64_shell 
	main!
hello world!
good symbol!
gDyld found, using dual hijack technique.
aϠ'
   pϠ'
      iϠ'
         sϠ'
            :Ϡ'
                Ϡ'
                  4Ϡ'
                     4Ϡ'
                        8Ϡ'
                           0Ϡ'
                              7Ϡ'
                                 7Ϡ'
                                    8Ϡ'
                                       3Ϡ'
                                          3Ϡ'
                                             6Ϡ'

Ϡ'
  cϠ'
     oϠ'
        nϠ'
           fϠ'
              iϠ'
                 gϠ'
                    :Ϡ'
                        Ϡ'
                          8Ϡ'
                             6Ϡ'
                                3Ϡ'
                                   6Ϡ'
                                      9Ϡ'
                                         4Ϡ'
                                            8Ϡ'
                                               4Ϡ'
                                                  9Ϡ'
                                                     6Ϡ'

Ϡ'
  bϠ'
     aϠ'
        sϠ'
           eϠ'
              :Ϡ'
                  Ϡ'
                    8Ϡ'
                       6Ϡ'
                          3Ϡ'
                             5Ϡ'
                                9Ϡ'
                                   0Ϡ'
                                      4Ϡ'
                                         0Ϡ'
                                            0Ϡ'
                                               0Ϡ'

Ϡ'
  sϠ'
     dϠ'
        yϠ'
           lϠ'
              dϠ'
                 :Ϡ'
                     Ϡ'
                       8Ϡ'
                          6Ϡ'
                             3Ϡ'
                                6Ϡ'
                                   5Ϡ'
                                      0Ϡ'
                                         6Ϡ'
                                            1Ϡ'
                                               1Ϡ'
                                                  2Ϡ'

Ϡ'
  EϠ'
     rϠ'
        rϠ'
           nϠ'
              oϠ'
                 :Ϡ'
                     Ϡ'
                       2Ϡ'

Ϡ'
  BϠ'
     uϠ'
        fϠ'
           fϠ'
              eϠ'
                 rϠ'
                    :Ϡ'
                        Ϡ'
                          4Ϡ'
                             4Ϡ'
                                8Ϡ'
                                   2Ϡ'
                                      1Ϡ'
                                         5Ϡ'
                                            8Ϡ'
                                               5Ϡ'
                                                  9Ϡ'
                                                     2Ϡ'

Ϡ'
  Analyzing Segments.
vϠ'
   mϠ'
      SϠ'
         pϠ'
            aϠ'
               cϠ'
                  eϠ'
                     :Ϡ'
                         Ϡ'
                           8Ϡ'
                              6Ϡ'
                                 0Ϡ'
                                    1Ϡ'
                                       6Ϡ'
                                          0Ϡ'

Ϡ'
  Rosetta.
TϠ'
   rϠ'
      aϠ'
         nϠ'
            sϠ'
               lϠ'
                  aϠ'
                     tϠ'
                        eϠ'
                           dϠ'
                              :Ϡ'
                                  Ϡ'
                                    tϠ'
                                       rϠ'
                                          uϠ'
                                             eϠ'

Ϡ'
  VϠ'
     MϠ'
        AϠ'
           lϠ'
              lϠ'
                 oϠ'
                    cϠ'
                       aϠ'
                          tϠ'
                             eϠ'
                                 Ϡ'
                                   RϠ'
                                      eϠ'
                                         tϠ'
                                            :Ϡ'
                                                Ϡ'
                                                  0Ϡ'

...etc...
...etc...
...etc...
...etc...

Has local variables.
Setting up locals.
Setting up locals.
Setting up locals.
Setting up locals.
Has local variables.
Setting up locals.
Has local variables.
Setting up locals.
Setting up locals.
Setting up locals.
Setting up locals.
Setting up locals.
Setting up locals.
Setting up locals.
Setting up locals.
Notifying.
Initializing
H?P?
    a?P?
        n?P?
            d?P?
                l?P?
                    e?P?
                        :?P?
                             ?P?
                                9?P?
                                    0?P?
                                        1?P?
                                            4?P?
                                                3?P?
                                                    8?P?
                                                        7?P?
                                                            7?P?
                                                                7?P?
                                                                    6?P?

?P?
   V?P?
       M?P?
```